tests/endpoints/echo/client/client.go

```diff
@@ -68,11 +68,12 @@ func DoWithHeaders(url, method, message string, headers map[string]string) ([]by
 		return nil, fmt.Errorf("create request error: %v", err)
 	}
 
-	if method == "POST" || method == "PATCH" || method == "PUT" {
+	if method != "GET" {
```

Comment:
`OPTIONS`? Make sure our CORS tests don't use this

Reply:
This method is not for OPTIONS. Anyway, change the condition for only checking if the body is empty.